socorro/unittest/processor/test_processor_app.py

```diff
@@ -170,7 +170,7 @@ def mocked_save_raw_and_processed(*_):
         with pytest.raises(PolyStorageError):
             pa.transform('mycrashid')
 
-        config.logger.warning.assert_called_with(
+        config.logger.warning.assert_any_call(
```

Comment:
There are now two logging calls, so we have to switch this to `assert_any_call` so it looks at all of them rather than the most recent.